[FIX] PERF en account dashboard account_balance_general

maq-adhoc opened the pull request.

maq-adhoc committed:
[FIX] PERF en account dashboard account_balance_general

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/807.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/807)

maq-adhoc committed:
[PERF] account_accountant_ux: fix dashboard values
[PERF] account_tax_settlement: delete values in tax dashboard

jjscarafia committed:
[IMP] accountant_ux: better performance, avoid join

jjscarafia committed:
[FIX] tax_settlement: fix dashboard view

Arreglamos la vista de manera simplem, sacamos el if porque sin all allow per line no lo usamos nunca (o no tenemos funiconalidad basicamente).

jjscarafia commented:
@roboadhoc r+

roboadhoc commented:
@maq-adhoc @jjscarafia because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

jjscarafia commented:
@roboadhoc rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
